modules/ROOT/pages/ai-openai.adoc

```diff
@@ -0,0 +1,204 @@
+= OpenAI ChatGPT integration guide
```

Comment:
@MitchC1999 I wanted to avoid naming a specific version like GPT-3 or GPT-4, but I'm not sure if calling the product ChatGPT is the correct usage of the term here.

Reply:
I'd say it's fine how it is